### PR TITLE
Add favicon to settings.html and system_info.html

### PR DESCRIPTION
**Pull request by JeffResc**

### CODE REVIEW

File: templates/settings.html [hunk 1]
```diff
@@ -8,6 +8,26 @@
     {% else %}
         <title>Screenly OSE</title>
     {% endif %}
+        <link href="/static/favicons/apple-touch-icon-57x57.png" rel="apple-touch-icon-precomposed" sizes="57x57"/>
+    <link href="/static/favicons/apple-touch-icon-114x114.png" rel="apple-touch-icon-precomposed" sizes="114x114"/>
```
Comment (vpetersson) [inline templates/settings.html]: Please fix the indentation.

File: templates/system_info.html [hunk 2]
```diff
@@ -8,6 +8,26 @@
     {% else %}
         <title>Screenly OSE</title>
     {% endif %}
+        <link href="/static/favicons/apple-touch-icon-57x57.png" rel="apple-touch-icon-precomposed" sizes="57x57"/>
+    <link href="/static/favicons/apple-touch-icon-114x114.png" rel="apple-touch-icon-precomposed" sizes="114x114"/>
```
Comment (vpetersson) [inline templates/system_info.html]: Same.

**Comment by vpetersson** [comment]
Thanks @JeffResc - just some minor formatting.

**Comment by JeffResc** [comment]
@vpetersson My apologies, editing online and the website didn't display the indentations correctly. Should be all set now.

**Comment by vpetersson** [comment]
No worries. Thanks, @JeffResc. merging.

**Pull request merged**